tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -597,6 +597,16 @@ internal static class FeatureFlags
             /// </summary>
             public const string OpenTelemetryEnabled = "DD_TRACE_OTEL_ENABLED";
 
+            /// <summary>
+            /// Enables the use of the <see cref="ISpan.OperationName"/> being set to the legacy value.
+            /// This flag defaults to <see langword="false"/> and is intended to allow beta users of OpenTelemetry support
+            /// to toggle on to give them time to upgrade to the new format. This additionally requires that
+            /// the <c>ActivitySource</c> has a <c>Name</c> property which was introduced in .NET 5 and/or v5 of
+            /// <c>System.Diagnostics</c> library.
+            /// Note: This feature flag may be dropped when our OpenTelemetry support becomes generally available.
+            /// </summary>
+            public const string OpenTelemetryLegacyOperationNameEnabled = "DD_TRACE_OTEL_LEGACY_OPERATION_NAME_ENABLED";
```

Comment:
Not urgent but you also need to do a PR to add it [here](https://github.com/DataDog/dd-go/blob/prod/trace/apps/tracer-telemetry-intake/telemetry-payload/static/config_norm_rules.json).

Reply:
Thanks will do that!